docs/data/date-pickers/localization/LocalizedDatePicker.js

```diff
@@ -13,13 +13,7 @@ import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { TimePicker } from '@mui/x-date-pickers/TimePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 
-const localeMap = {
-  en: undefined,
-  fr: frLocale,
-  de: deLocale,
-  ru: ruLocale,
-  ar: arSaLocale,
-};
+const locales = ['en', 'fr', 'de', 'ru', 'ar-sa'];
```

Comment:
I misused the localization api of dayjs 
We are supposed to pass the locale string to the adapter, not the locale object